resources/dicts/thoughts/dead/darkforest/general.json

```
@@ -60,10 +46,100 @@
             "Is thinking about walking in the dreams of r_c",
             "Watches r_c with malice in {PRONOUN/m_c/poss} eyes",
             "Is whispering in the ear of r_c",
-            "Is watching r_c with a sly smile"
+            "Is watching r_c with a sly smile",
+            "Wonders what r_c would think if {PRONOUN/r_c/subject} saw {PRONOUN/m_c/object} now"
         ],
         "random_living_status": [
             "living"
         ]
+    },
+    {
+        "id": "dark_forest_kind",
+        "thoughts": [
+            "Wishes {PRONOUN/m_c/subject} could go to StarClan",
+            "Feels regret for {PRONOUN/m_c/poss} actions",
+            "Is curled up in a ball crying",
+            "Sits longingly at the StarClan border",
+            "Is whimpering pathetically",
+            "Scarcely avoided a confrontation with a shadowy cat",
+            "Bought {PRONOUN/m_c/self} some time to hide after talking {PRONOUN/m_c/poss} way out of training",
+            "Is wondering if everything was really worth it...",
+            "Is fleeing from another member of the Dark Forest",
+            "Wants nothing more then to fade away...",
+            "Wonders where everything went wrong",
+            "Is begging a shadowy cat for mercy",
+            "Has lost all hope",
+            "Wonders if there's anyone here {PRONOUN/m_c/subject} can actually trust",
+            "Tried crossing the border into StarClan to no avail"
+        ],
+        "main_trait_status": [
+            "careful",
+            "compassionate",
+            "loving",
+            "gloomy",
+            "nervous",
+            "playful",
+            "responsible",
+            "sincere",
+            "strange",
+            "thoughtful"
+        ]
+    },
+    {
+        "id": "dark_forest_evil",
+        "thoughts": [
+            "Is fighting with another member of the Dark Forest",
+            "Wishes {PRONOUN/m_c/subject} could hurt those who have wronged {PRONOUN/m_c/object}",
+            "Wants to unleash mayhem on the cats who hurt {PRONOUN/m_c/object} in life",
+            "Is scheming",
+            "Found a living cat that {PRONOUN/m_c/subject} could manipulate",
+            "Found a living cat that {PRONOUN/m_c/subject} {VERB/m_c/hope/hopes} will help {PRONOUN/m_c/poss} plans come to fruition",
```

```suggestion
            "Found a living cat that, hopefully, will help {PRONOUN/m_c/poss} plans come to fruition",
```
sentence felt stilted